### PR TITLE
Switch font-awesome for material for main icons

### PR DESCRIPTION
**Pull request by JabX**
Forms, dates and error center.
There's more but they were the ones that I noticed the most.

### CODE REVIEW

File: src/application/error-center/index.js [hunk 1]
```diff
@@ -47,12 +47,12 @@ const errorCenter = {
...
             <div data-focus='error-center'>
                 <div data-focus='error-counter'>
-                    <i className="fa fa-times-circle"></i>{errorLength}
+                    <i className='material-icons' style={{cursor: 'pointer', fontSize: '28px', padding: '15px 5px 5px 5px'}}>error</i>{errorLength}
                 </div>
                 <div data-focus='error-actions'>
-                    <i className='fa fa-refresh' onClick={()=>{window.location.reload(); }}></i>
-                    <i className={`fa fa-arrow-circle-o-${isErrorsVisible ? 'up' : 'down'}`} onClick={this._toggleVisible}></i>
-                    <i className='fa fa-trash-o' onClick={()=>{this.setState({errors: []}); }}></i>
+                    <i className='material-icons' style={{cursor: 'pointer', fontSize: '36px', padding: '10px'}} onClick={()=>{window.location.reload(); }}>refresh</i>
```
Comment (pierr) [inline src/application/error-center/index.js]: why `fontSize` ?

**Comment by pierr** [comment]
@JabX Thanks!

**Comment by pierr** [comment]
@Tommass  can you check the font size?

**Comment by Tommass** [comment]
Mortel !
Thanks @JabX

**Pull request merged**